docs/src/standard_results/depth_surfaces.md

```diff
@@ -0,0 +1,44 @@
+# Modelled depth prediction surfaces
+
+This exports the modelled depth prediction surfaces from within RMS.
+These surfaces typically represent the final surface set generated during a structural
+modeling workflow (after well conditioning), and serve as the framework for constructing the grid.
+
+:::{table} Current
+:widths: auto
+:align: left
+
+| Field | Value |
+| --- | --- |
+| Version | NA |
+| Output | `share/results/maps/depth_predictions/surfacename.gri` |
+:::
+
+## Requirements
+
+- RMS
+- depth surfaces stored in a horizon folder within RMS
+
+The depth surfaces should be located within a horizon folder in RMS.
+The simplified export function will use the RMS API behind the scene to automatically
+identify non-empty horizons from the folder, and export them using `fmu.dataio`.
+
+
+## Usage
+
+```{eval-rst}
+.. autofunction:: fmu.dataio.export.rms.depth_prediction_surfaces.export_depth_prediction_surfaces
+```
+
+## Result
+
+The surfaces from within the horizon folder will be exported as 'irap_binary'
```

Comment:
Do we want to state this explicitly? Or, rather, do we want to be able to change this? Currently, it sort of looks like an invitation to a discussion around which folder and filenames that should be used - but ideally nobody is going to relate to that.